include/interfaces/interface.h

```diff
@@ -327,10 +328,13 @@ class Interface : public ParsedFiniteElement<dim,spacedim>
   std::string str_diff_comp;
   std::vector<unsigned int> _diff_comp;
 
+  mutable const typename LAC::VectorType *old_solution;
```

Comment:
This should be an object, not a pointer. If it's a pointer, it points to the same thing as solution...